research/nonequilibrium-self-assembly/index.md

```diff
@@ -8,14 +8,16 @@ title: Non-equilibrium simulations
 {% capture text %}
 
 <!-- Add text -->
+Nanoparticles or colloidal dispersions play a vital role in our daily lives. From paints to catalysts, can be produced using colloidal dispersions. If a dispersion has different types of particles, various aspects can be controlled to make it self-assemble into desired structures. One such aspect we are controlling in this study is solvent evaporation. Solvent evaporation allows us to create self-assembled coatings and supraparticles cheaply and efficiently. However, it is hard to know what combination of parameters, such as evaporation rate, particle sizes, initial film or droplet length, etc., would result in the desired structure when using the solvent evaporation method.
```

Comment:
`From paints to catalysts, can be produced using colloidal dispersions.` This sentence is clunky. Maybe something like: 
`Various useful products, from paints to catalysts, can be produced using colloidal dispersions.`